module.php

```diff
@@ -2,14 +2,15 @@
 use Psr\Container\ContainerInterface;
 use \RebelCode\Bookings\WordPress\Module\WpBookingsFrontUi;
 
-define('RC_WP_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR', 'modules/rcmod-wp-bookings-front-ui');
-define('RC_WP_BOOKINGS_FRONT_UI_MODULE_DIR', __DIR__);
-define('RC_WP_BOOKINGS_FRONT_UI_MODULE_CONFIG', RC_WP_BOOKINGS_FRONT_UI_MODULE_DIR . '/config.php');
-define('RC_WP_BOOKINGS_FRONT_UI_MODULE_KEY', 'wp_bookings_front_ui');
+define('WP_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR', 'modules/rcmod-wp-bookings-front-ui');
+define('WP_BOOKINGS_FRONT_UI_MODULE_DIR', __DIR__);

```

Comment:
@mecha, what do you think about this naming of constants here?

Reply:
Looks normal, even if the names are long and verbose. But that's not necessarily a bad thing.

Reply:
@mecha, the `WP_` prefix is for WP constants.

Reply:
Or WaterPark!

True though. Better to use `RC_` or `RCMOD_`.

Reply:
Agreed. I think I prefer the way they used to be. What prompted the change anyway, @romalytvynenko?